assets/pages/new-contributions-landing/components/ContributionUsTicker.jsx

```diff
@@ -0,0 +1,120 @@
+// @flow
+
+// ----- Imports ----- //
+import React, { Component } from 'react';
+
+// ---- Types ----- //
+type StateTypes = {|
+  totalSoFar: number,
+  goal: number,
+|}
+
+type PropTypes = {}
+
+
+// ---- Helpers ----- //
+
+const getInitialTickerValues = (): Promise<{| totalSoFar: number, goal: number |}> =>
+  fetch('https://interactive.guim.co.uk/docsdata-test/1ySn7Ol2NQLvvSw_eAnVrPuuRnaGOxUmaUs6svtu_irU.json')
+    .then(resp => resp.json())
+    .then((data) => {
+      const totalSoFar = parseInt(data.sheets.Sheet1[0].total, 10);
+      const goal = parseInt(data.sheets.Sheet1[0].goal, 10);
+
+      return { totalSoFar, goal };
+    });
+
+const percentageTotalAsNegative = (total: number, goal: number) => {
+  let percentage = ((total / goal) * 100) - 100;
+  if (percentage > 0) {
+    percentage = 0;
+  }
+  return percentage;
```

Comment:
minor: could use a ternary and a const here to avoid the let

```suggestion
const percentage = ((total / goal) * 100) - 100;
return percentage > 0 ? 0 : percentage;
```